dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/system/SystemInfo.java

```diff
@@ -118,6 +118,10 @@
     private boolean encryption;
 
     private boolean emailConfigured;
+    
+    private String redisEnabled;
```

Comment:
Let's use *boolean* redisEnabled - easier to use in Java and consistent with other yes/no properties.